### PR TITLE
ioporder: use right name on reset

### PR DESCRIPTION
**Pull request by sgotti**
On reset a fixed string was used instead of the utility function `dt_iop_order_string`.

Without the patch:

Click Reset:
image

After reset:
image

With this patch, after clicking reset the right name (`v3.0 RAW`) is used.

### CODE REVIEW

File: src/libs/ioporder.c [hunk 1]
```diff
@@ -191,8 +191,8 @@ void gui_reset (dt_lib_module_t *self)
     dt_dev_pixelpipe_rebuild(darktable.develop);
 
     d->current_mode = DT_IOP_ORDER_V30;
-    if(d->widget)
-      gtk_label_set_text(GTK_LABEL(d->widget), _("v3.0"));
+    if(d->widget) gtk_label_set_text(GTK_LABEL(d->widget), _(dt_iop_order_string(DT_IOP_ORDER_V30)));
```
Comment (TurboGit) [inline src/libs/ioporder.c:194]: Maybe: `_(dt_iop_order_string(d->current_mode))` to ensure that if the line above is changed we do not forget to update this one. I'm wondering if somehow it is not what has happened here.

Comment (sgotti) [reply src/libs/ioporder.c]: @TurboGit Pushed new commit with updated and simplification

File: src/libs/ioporder.c [hunk 2]
```diff
@@ -120,23 +120,23 @@ void update(dt_lib_module_t *self)
     if(!found)
     {
       d->current_mode = DT_IOP_ORDER_CUSTOM;
-      gtk_label_set_text(GTK_LABEL(d->widget), _(dt_iop_order_string(DT_IOP_ORDER_CUSTOM)));
+      gtk_label_set_text(GTK_LABEL(d->widget), _(dt_iop_order_string(d->current_mode)));
     }
   }
   else if(kind == DT_IOP_ORDER_LEGACY)
```
Comment (TurboGit) [inline src/libs/ioporder.c:126]: and then the 3 blocks bellow can be rewritten as:

```
else 
{
    d->current_mode = kind;
    gtk_label_set_text(GTK_LABEL(d->widget), _(dt_iop_order_string(d->current_mode)));
}
```

**Review by TurboGit — APPROVED** [review]
Looks good to me now, thanks!

**Pull request merged**